src/main/resources/db/migration/V25__add_hakulomake_columns_to_haut_and_hakuehdot.sql

```diff
@@ -0,0 +1,146 @@
+alter table haut
+  add column hakulomake_id character varying,
+  add column hakulomake_kuvaus jsonb,
+  add column hakulomake_linkki jsonb;
+
+alter table haut_history
+  add column hakulomake_id character varying,
+  add column hakulomake_kuvaus jsonb,
+  add column hakulomake_linkki jsonb;
+
+alter table hakukohteet
+  add column hakulomake_id character varying,
+  add column hakulomake_kuvaus jsonb,
+  add column hakulomake_linkki jsonb,
+  add column eri_hakulomake boolean;
```

Comment:
Hakukohteella on kenttä kaytetaan_haun_aikataulua, joten tämä voisi olla vastaavasti kaytetaan_haun_hakulomaketta. Tai jos ei haluta, että boolean kääntyy toisin päin, niin sitten ei_kayteta_haun_hakulomaketta, eri_hakulomake_kuin_haulla tms.